shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/adapter/impl/ShardingSphereProxyClusterContainer.java

```diff
@@ -47,17 +45,18 @@ public final class ShardingSphereProxyClusterContainer extends DockerITContainer
     
     private final String scenario;
     
-    private final String module;
+    private final AdaptorContainerConfiguration adaptorContainerConfiguration;
```

Comment:
`adaptorContainerConfiguration` could be rename to `config`

Reply:
done